.github/workflows/get_image.yaml

```diff
@@ -13,6 +13,8 @@ on:
       image-with-tag:
         description: "A image with tag (from docker.properties) for the requested image"
         value: ${{ jobs.get-image.outputs.image-with-tag }}
+permissions:
+  contents: read
```

Comment:
I thought this was the default?
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
Depends on if the repo is configured in permissive or restrictive mode.
This ensures a lower level of access no matter the repo config.